src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/advices/ResourceExceptionHandler.java

```diff
@@ -20,17 +20,10 @@
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
 import uk.gov.hmcts.cmc.claimstore.appinsights.AppInsightsExceptionLogger;
-import uk.gov.hmcts.cmc.claimstore.exceptions.CallbackException;
-import uk.gov.hmcts.cmc.claimstore.exceptions.ClaimantLinkException;
-import uk.gov.hmcts.cmc.claimstore.exceptions.ConflictException;
-import uk.gov.hmcts.cmc.claimstore.exceptions.DefendantLinkingException;
-import uk.gov.hmcts.cmc.claimstore.exceptions.DocumentDownloadForbiddenException;
-import uk.gov.hmcts.cmc.claimstore.exceptions.ForbiddenActionException;
-import uk.gov.hmcts.cmc.claimstore.exceptions.InvalidApplicationException;
-import uk.gov.hmcts.cmc.claimstore.exceptions.NotFoundException;
-import uk.gov.hmcts.cmc.claimstore.exceptions.OnHoldClaimAccessAttemptException;
+import uk.gov.hmcts.cmc.claimstore.exceptions.*;
```

Comment:
please avoid .* imports, and revert above change.

Reply:
It wasn't me. Looks like IntelliJ did it.